sdk/tables/azure-data-tables/README.md

```diff
@@ -1,53 +1,57 @@
 # Azure Data Tables client library for Python
 
-Azure Data Tables is a NoSQL data storing service that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS. 
-Tables scale as needed to support the amount of data inserted, and allow for the storing of data with non-complex accessing. 
-Tables scale as needed to support the amount of data inserted, and allow for the storing of data with non-complex accessing. 
+Azure Data Tables is a NoSQL data storage service that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS.
+Tables scales as needed to support the amount of data inserted, and allow for the storing of data with non-complex accessing.
 The Azure Data Tables client can be used to access Azure Storage or Cosmos accounts.
 
 Common uses of Azure Data Tables include:
 
 * Storing structured data in the form of tables
 * Quickly querying data using a clustered index
 
-[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk) | [Package (PyPI)](https://pypi.org) | [API reference documentation](https://aka.ms/azsdk/python/tables/docs) | [Product documentation](https://docs.microsoft.com/azure/storage/) | [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk)
+[Source code](source_code) | [Package (PyPI)](Tables_pypi) | [API reference documentation](Tables_ref_docs) | [Product documentation](Tables_product_doc) | [Samples](Tables_samples)
 
 ## Getting started
 
 ### Prerequisites
 * Python 2.7, or 3.5 or later is required to use this package.
-* You must have an [Azure subscription](https://azure.microsoft.com/free/) and an
-[Azure storage account](https://docs.microsoft.com/azure/storage/common/storage-account-overview) to use this package
- or you must have a [Azure Cosmos Account](https://docs.microsoft.com/azure/cosmos-db/account-overview).
+* You must have an [Azure subscription](azure_subscription) and an
+[Azure storage account](azure_storage_account) to use this package
+ or an [Azure Cosmos Account](azure_cosmos_account).
 
 ### Install the package
-Install the Azure Data Tables client library for Python with [pip](https://pypi.org/project/pip/):
+Install the Azure Data Tables client library for Python with [pip](pip_link):
 
 ```bash
 pip install --pre azure-data-tables
 ```
 
 ### Create a storage account
```

Comment:
I'd try to make this and the next section listed in a way that's a bit clearer to show that it's either one or the other but not both just like in the credentials section

Reply:
These headings: Create a storage account, Creating a Cosmos DB, Create the client, should be one heading level below "Instal the Package": so h4